src/views/catalog/wizard/components/WizardFooter.tsx

```diff
@@ -21,15 +21,20 @@ import { clearSessionStorageVM, useWizardVMContext } from '../../utils/WizardVMC
 
 import { WizardNoBootModal } from './WizardNoBootModal';
 
-export const WizardFooter: React.FC<{ namespace: string }> = ({ namespace }) => {
+export const WizardFooter: FC<{ namespace: string }> = ({ namespace }) => {
   const history = useHistory();
   const { t } = useKubevirtTranslation();
-  const { tabsData, disableVmCreate, loaded: vmContextLoaded } = useWizardVMContext();
+  const {
+    tabsData,
+    updateTabsData,
+    disableVmCreate,
+    loaded: vmContextLoaded,
+  } = useWizardVMContext();
   const { isBootSourceAvailable, loaded: bootSourceLoaded } = useWizardSourceAvailable();
   const { createVM, error, loaded: vmCreateLoaded } = useWizardVmCreate();
   const { createModal } = useModal();
 
-  const [startVM, setStartVM] = React.useState(true);
+  const [startVM, setStartVM] = useState<boolean>(tabsData?.startVM || true);
```

Comment:
tabsData?.startVM || true will evaluate to true all the time:
tabsData?.startVM === true => true || true => true
tabsData?.startVM === false => false || true => true
you can remove the || true part as it will always should have the value carried from the drawer